INSTALL.md

```diff
@@ -0,0 +1,90 @@
+# Launch a Network
+
+The following are instructions on setting up a group of Core validators in order to setup a new or join an existing network.
+
+Kin Foundation are currently using AWS for managing its infrastructure, though other cloud providers are good as well.
```

Comment:
Kin Ecosystem

Reply:
I'm referring to the Kin NPO, the "ecosystem" can be confusing in the case and might be misinterpreted to the entire ecosystem.